src/frontends/pytorch/src/op/adaptive_avg_pool3d.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_adaptive_avg_pool3d(NodeContext& context) {
+    auto const_tile_params = context.mark_node(opset10::Constant::create(element::i32, Shape{5}, {1, 1, 1, 1, 1}));
```

Comment:
please check that node has two inputs. Can you please add such check?

Reply:
That seems like a common issue in all translators, I suggest doing such changes separately from frontend merge to master